vertx-pg-client/src/main/java/io/vertx/pgclient/PgConnectOptions.java

```diff
@@ -94,6 +96,15 @@ public static PgConnectOptions fromEnv() {
   public static final String DEFAULT_PASSWORD = "pass";
   public static final int DEFAULT_PIPELINING_LIMIT = 256;
   public static final SslMode DEFAULT_SSLMODE = SslMode.DISABLE;
+  public static final Map<String, String> DEFAULT_PROPERTIES = new HashMap<>();
```

Comment:
make this map unmodifiable in the static block

Reply:
fixed in https://github.com/eclipse-vertx/vertx-sql-client/pull/344/commits/b31529d3d6c34f6e89fac44903415e7cb2551b02.